### PR TITLE
modernized-scrollbar-and-moved-functions-matthew

### LINKED ISSUE #107: Create modernized scrollbar
Task
- Create modernized scrollbar as discussed at the meeting

Deliverables
- New and improved scrollbar for the dashboard

### LINKED ISSUE #108: Move GitHub functions from index.js to github-api.js
Task
- Move GitHub functions from index.js file to github-api.js file inside the js folder

Deliverables
- Separated GitHub functions in the github-api.js file

### PR DESCRIPTION
**Pull request by blankss**
## Feature/Change Description
- Moved the GitHub API calls to their own file.
- Modernized scrollbar.

## Developer Contributions
- @blankss 

## Dependencies
- N/A

Resolves #107 
Resolves #108

### CODE REVIEW

File: src/js/github-api.js [hunk 1]
```diff
@@ -20,3 +22,63 @@ export async function isProviderTokenValid(token) {
...
+    if (!response.ok) {
+      throw new Error('Failed to fetch pull requests.');
+    }
+
+    const data = await response.json();
+    return data;
+
+  }
+  catch (error) {
+    console.log(error);
```
Comment (mottur) [inline src/js/github-api.js:52]: Try to avoid leaving ```console``` statements in when merging to release.

**Review by mottur — APPROVED** [review]
Modernized scrollbar, moved GitHub API calls to their own file. Approved.

File: src/js/github-api.js (hunk 1 above)
Comment (blankss) [reply src/js/github-api.js]: Replaced with console.error similar to David's function.

**Review by mottur — APPROVED** [review]
Implemented modernized scrollbar. Moved GitHub API calls to new file. Approved.

**Pull request merged**